DESCRIPTION

```diff
@@ -16,36 +16,31 @@ Authors@R: c(
     person(given = "Bedford Freeman & Worth Pub Grp LLC DBA Macmillan Learning", 
            role = c("cph"))
     )
-Description: Tokenize words into morphemes (the smallest unit of meaning).
+Description: Tokenize text into morphemes.
```

Comment:
They've dinged me before for Descriptions not being descriptive enough. I'd keep the parenthetical, plus maybe expand a little. I notice, for example, that https://cran.r-project.org/package=wordpiece was more detailed. Maybe word it as "using the morphemepiece algorithm" even though we don't (yet) have a paper to link to.